corehq/apps/cloudcare/views.py

```diff
@@ -468,12 +470,32 @@ def put(self, request, *args, **kwargs):
 @login_and_domain_required
 def report_formplayer_error(request, domain):
     data = json.loads(request.body)
-    metrics_counter('commcare.formplayer.browser_error_reports', tags={
-        'action': data.get('action'),
-        'statusText': data.get('statusText'),
-        'state': data.get('state'),
-        'status': data.get('status'),
-        'domain': domain,
-    })
-    notify_error(message='Formplayer error reported from the browser', details=data)
+    error_type = data.get('type')
+    if error_type == 'webformsession_request_failure':
+        metrics_counter('commcare.formplayer.webformsession_request_failure', tags={
+            'action': data.get('action'),
+            'statusText': data.get('statusText'),
+            'state': data.get('state'),
+            'status': data.get('status'),
+            'domain': domain,
+        })
+        notify_error(message='Formplayer: request failure in web form session', details=data)
+    elif error_type == 'show_error_notification':
+        message = data.get('message')
+        metrics_counter('commcare.formplayer.show_error_notification', tags={
+            'message': _message_to_tag_value(message or 'no_message'),
+            'domain': domain,
+        })
+        notify_error(message=f'Formplayer: showed error to user: {message}', details=data)
+    else:
+        metrics_counter('commcare.formplayer.unknown_error_type', tags={
+            'domain': domain,
+        })
+        notify_error(message=f'Formplayer: unknown error type', details=data)
     return JsonResponse({'status': 'ok'})
+
+
+def _message_to_tag_value(message, allowed_chars=string.ascii_lowercase + '_'):
+    message_tag = '_'.join(unidecode(message).split(' ')[:4]).lower()
```

Comment:
I'm not following the `:4`, what's that?

Reply:
Yeah sorry not the easiest to follow.

Added a docstring with examples to help make it clearer

Reply:
thanks!